.github/merge_rules.yaml

```diff
@@ -236,6 +236,21 @@
   - Lint
   - pull
 
+- name: XPU ATen
+  patterns:
+  - third_party/xpu.txt
+  approved_by:
+  - EikanWang
+  - jgong5
+  - gujinghui
+  - alband
+  - malfet
+  - atalman
```

Comment:
I don't think this is needed here, as all 3 are already covered by different rules
```suggestion
```